packages/target-tauri/src-tauri/src/menus/main_menu.rs

```diff
@@ -0,0 +1,278 @@
+                                    handle,
+                                    "dev_tools",
+                                    "Open Developer Tools",
+                                    true,
+                                    None::<&str>,
+                                )?,
+                                &MenuItem::with_id(
+                                    handle,
+                                    "log_folder",
+                                    "Open the Log Folder",
+                                    true,
+                                    None::<&str>,
+                                )?,
+                                &MenuItem::with_id(
+                                    handle,
+                                    "current_log_file",
+                                    "Open Current Log File",
+                                    true,
+                                    None::<&str>,
+                                )?,
+                            ],
+                        )?,
+                    ],
+                )?,
+                &Submenu::with_items(
+                    handle,
+                    "Help",
+                    true,
+                    &[
+                        &MenuItem::with_id(handle, "help", "Help", true, None::<&str>)?,
+                        &MenuItem::with_id(
+                            handle,
+                            "keybindings",
+                            "Keybindings",
+                            true,
+                            None::<&str>,
+                        )?,
+                        &MenuItem::with_id(
+                            handle,
+                            "learn",
+                            "Learn more about Delta Chat",
+                            true,
+                            None::<&str>,
+                        )?,
+                        &MenuItem::with_id(
+                            handle,
+                            "contribute",
+                            "Contribute on Github",
+                            true,
+                            None::<&str>,
+                        )?,
+                        &PredefinedMenuItem::separator(handle)?,
+                        &MenuItem::with_id(
+                            handle,
+                            "report",
+                            "Report an Issue",
+                            true,
+                            None::<&str>,
+                        )?,
+                        &MenuItem::with_id(
+                            handle,
+                            "about",
+                            "About Delta Chat",
+                            true,
+                            None::<&str>,
+                        )?,
+                    ],
+                )?,
+            ],
+        )
+    });
+    builder.on_menu_event(|app, event| {
+        match event.id().as_ref() {
+            "settings" => {
```

Comment:
maybe you could make constants or a string enum for these, to prevent typos between the two instances of the string.
enum would especially make sense, because then we could have a match statement that ensures all cases are handled.